de.dlr.sc.overtarget.language.tests/src/de/dlr/sc/overtarget/language/tests/OvertargetOutputConfigurationProviderTest.java

```diff
@@ -12,59 +12,50 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
 
 import java.util.Set;
 
-import org.eclipse.xtext.generator.IFileSystemAccess;
 import org.eclipse.xtext.generator.OutputConfiguration;
 import org.junit.Test;
 
 import de.dlr.sc.overtarget.language.generator.OvertargetOutputConfigurationProvider;
 /**
- * Testcase for OvertargetOutputConfigurationProvider
+ * Test Case for OvertargetOutputConfigurationProvider
  * @author kuja_tj
  *
  */
 public class OvertargetOutputConfigurationProviderTest {
 
 	@Test
 	public void testGetOutputConfigurations() {
-		final int NUMBER_OF_OUTPUTS = 2;
 
-		final String GENERATOR_OUTPUT_ID_OVERTARGET = "de.dlr.sc.overtarget.output";
-		final String GENERATOR_OUTPUT_FOLDER_CONCEPT = "./";
-		final String GENERATOR_OUTPUT_DESCRIPTION_CONCEPT = "Xtend Overtarget Serialzation Output";
-
-		final String GENERATOR_OUTPUT_ID_DEFAULT = IFileSystemAccess.DEFAULT_OUTPUT;
-		final String GENERATOR_OUTPUT_FOLDER_DEFAULT = "./default-gen";
-		final String GENERATOR_OUTPUT_DESCRIPTION_DEFAULT = "Xtend Default Output";
-		
 		OvertargetOutputConfigurationProvider oocp = new OvertargetOutputConfigurationProvider();
 		Set<OutputConfiguration> outputConfigurations = oocp.getOutputConfigurations();
 		
-		assertEquals("The number of Output Configurations is correct", NUMBER_OF_OUTPUTS, outputConfigurations.size());

```

Comment:
The constant NUMBER_OF_OUTPUTS I would have left in

Reply:
Fixed

Reply:
@SaMuellerDLR Fix the change to NUMBER_OF_OUTPUTS. Ready to check.